manuscript/recipes/duplicati.md

```diff
@@ -0,0 +1,115 @@
+hero: Duplicati - Yet another option to backup your exciting stuff. It's good to have options.
+
+# Duplicati
+
+Intro
```

Comment:
Replace "Intro" with a brief headline. Here's a good one you could copy/paste/adapt: https://github.com/geek-cookbook/geek-cookbook/blob/master/manuscript/recipes/elkarbackup.md#L5

Reply:
I'm struggling to figure out what to do here. My creative brain is on leave today.

Reply:
Added a headline and proper attribution